task_2/__init__.py

```diff
@@ -0,0 +1 @@
+from .reverse_string import reverse_string, NonStringInputError
```

Comment:
Github itself shows you that there is something not up to standards

![image](https://github.com/user-attachments/assets/49f14196-d202-43ed-9749-568c3f2c6470)

https://www.flake8rules.com/rules/W292.html
It is recommended to have an empty line at the end of each file

Reply:
I added an empty line and I'll acquaint with other rules on the link, thank you for resource!